### ISSUE TITLE
Can't save feedItem and association with feedcategory.

### ISSUE THREAD

**Issue by hirakiuc** [hirakiuc/HatebuFeed#1]
Realm raise error like this.

```
2015-07-20 23:40:02.678 xctest[45791:4604551] *** Terminating app due to uncaught exception 'RLMException', reason: 'Can't mutate a persisted array outside of a write transaction.'
```

**Comment by hirakiuc** [comment]
[Here](https://github.com/hirakiuc/HatebuFeed/blob/master/HatebuFeed/BaseFeedRequest.swift#L77)

**Comment by hirakiuc** [comment]
fixed with commit (111ec4bc900a94629b524b1c11606dd103d65bc4).
